mw/RegExp.d.ts

```diff
@@ -0,0 +1,21 @@
+declare global {
+    namespace mw {
+        namespace RegExp {
+            /**
+             * Escape string for safe inclusion in regular expression
+             *
+             * The following characters are escaped:
+             *
+             *     \ { } ( ) | . ? * + - ^ $ [ ]
+             *
+             * @since 1.26
+             * @static
+             * @param {string} str String to escape
+             * @return {string} Escaped string
+             */
+            function escape(str: string): string;
```

Comment:
It's deprecated right? See the note near the bottom of https://doc.wikimedia.org/mediawiki-core/master/js/source/util.html.

This should have a `@deprecated` annotation suggesting use of mw.util.escapeRegExp instead.